pandas/io/parsers.py

```diff
@@ -167,9 +168,12 @@
     to preserve and not interpret dtype.
     If converters are specified, they will be applied INSTEAD
     of dtype conversion.
-engine : {{'c', 'python'}}, optional
-    Parser engine to use. The C engine is faster while the python engine is
-    currently more feature-complete.
+engine : {{'c', 'python', 'pyarrow'}}, optional
+    Parser engine to use. The C and pyarrow engines are faster, while the python engine
+    is currently more feature-complete. The pyarrow engine requires ``pyarrow`` >= 0.15
+    as a dependency however.
+    .. versionchanged:: 1.1
```

Comment:
```suggestion

    .. versionchanged:: 1.1
```

(added blank line above the versionchanged line)